migrations/educandu-2023-04-03-01-initial-schema.js

```diff
@@ -0,0 +1,260 @@
+      }
+    ]);
+    await createCollectionWithIndexes('requestLimitRecords', [
+      {
+        name: '_idx_expiresOn_',
+        key: { expiresOn: 1 },
+        expireAfterSeconds: 0
+      }
+    ]);
+    await createCollectionWithIndexes('roomInvitations', [
+      {
+        name: '_idx_expiresOn_',
+        key: { expiresOn: 1 },
+        expireAfterSeconds: 0
+      },
+      {
+        name: '_idx_roomId_email_',
+        key: { email: 1, roomId: 1 },
+        unique: true
+      },
+      {
+        name: '_idx_token_',
+        key: { token: 1 },
+        unique: true
+      }
+    ]);
+    await createCollectionWithIndexes('rooms', [
+      {
+        name: '_idx_owner_',
+        key: { owner: 1 }
+      },
+      {
+        name: '_idx_created_by_',
+        key: { createdBy: -1 }
+      },
+      {
+        name: '_idx_updated_by_',
+        key: { updatedBy: -1 }
+      },
+      {
+        name: '_idx_members_user_id_',
+        key: { 'members.userId': 1 }
+      },
+      {
+        name: '_idx_members_user_id_desc_',
+        key: { 'members.userId': -1 }
+      }
+    ]);
+    await createCollectionWithIndexes('sessions', [
+      {
+        name: '_idx_expires_',
+        key: { expires: 1 },
+        expireAfterSeconds: 0
+      }
+    ]);
+    await createCollectionWithIndexes('settings', []);
+    await createCollectionWithIndexes('storagePlans', [
+      {
+        name: '_idx_name',
+        key: { name: 1 },
+        unique: true
+      }
+    ]);
+    await createCollectionWithIndexes('tasks', [
+      {
+        name: '_idx_batch_id_',
+        key: { batchId: 1 }
+      },
+      {
+        name: '_idx_batch_id_processed_',
+        key: { batchId: 1, processed: 1 }
+      },
+      {
+        name: '_idx_task_id_processed_',
+        key: { taskId: 1, processed: 1 }
+      }
+    ]);
+    await createCollectionWithIndexes('users', [
+      {
+        name: '_idx_email_accountClosedOn_',
+        key: { email: 1, accountClosedOn: 1 },
+        unique: true,
+        partialFilterExpression: { $and: [{ email: { $type: 'string' } }, { accountClosedOn: null }] }
+      },
+      {
+        name: '_idx_expiresOn_',
+        key: { expiresOn: 1 },
+        expireAfterSeconds: 0
+      },
+      {
+        name: '_idx_verificationCode_',
+        key: { verificationCode: 1 },
+        unique: true,
+        partialFilterExpression: { verificationCode: { $type: 'string' } }
+      }
+    ]);
+  }
+
+  async down() {
+    await this.db.dropCollection('batches');
+    await this.db.dropCollection('comments');
+    await this.db.dropCollection('documentOrders');
+    await this.db.dropCollection('documentRevisions');
+    await this.db.dropCollection('documents');
+    await this.db.dropCollection('externalAccounts');
+    await this.db.dropCollection('locks');
+    await this.db.dropCollection('mediaLibraryItems');
+    await this.db.dropCollection('migrations');
+    await this.db.dropCollection('notifications');
```

Comment:
We added 2 collections since we last reset the DB state, btw 😳  `mediaLibraryItems` and `notifications`

Reply:
:+1: